### PR TITLE
docs: use new `nuxi module add` command in installation

### PR DESCRIPTION
**Pull request by danielroe**
This updates the documentation to use the [`nuxi module add` command](https://github.com/nuxt/cli/pull/197) which should simplify docs a bit and also improve user experience as there's no need to add to `nuxt.config` manually.

It's documented [here](https://nuxt.com/docs/api/commands/module#nuxi-module-add).

I may have missed a few spots in the documentation as I'm doing this across the modules ecosystem assisted by the power of regular expressions ✨, so I'd appreciate a review 🙏

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,19 +14,19 @@ Try it on [StackBlitz](https://stackblitz.com/edit/nuxt-svgo-playground?file=nux
 ### Using `npm`
```
Comment (danielroe) [inline README.md:15]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -14,19 +14,19 @@ Try it on [StackBlitz](https://stackblitz.com/edit/nuxt-svgo-playground?file=nux
...
 
 ```sh
-npm install nuxt-svgo --save-dev
+npx nuxi@latest module add nuxt-svgo
 ```
 
 ### Using `yarn`
 
 ```sh
```
Comment (danielroe) [inline README.md:22]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -14,19 +14,19 @@ Try it on [StackBlitz](https://stackblitz.com/edit/nuxt-svgo-playground?file=nux
...
-yarn add nuxt-svgo -D
+npx nuxi@latest module add nuxt-svgo
 ```
 
 ### Using `pnpm`
 
 ```sh
```
Comment (danielroe) [inline README.md:28]:
```suggestion
```

**Review by cpsoinos — APPROVED** [review]
Lgtm - thanks!

**Pull request merged**